src/03-modality-agnostic-files.md

```diff
@@ -26,8 +26,8 @@ Every dataset MUST include this file with the following fields:
 | HowToAcknowledge   | OPTIONAL              | [string][]               | Text containing instructions on how researchers using this dataset should acknowledge the original authors. This field can also be used to define a publication that should be cited in publications that use the dataset.                            |
 | Funding            | OPTIONAL              | [array][] of [strings][] | List of sources of funding (grant numbers).                                                                                                                                                                                                           |
 | EthicsApprovals    | OPTIONAL              | [array][] of [strings][] | List of ethics committee approvals of the research protocols and/or protocol identifiers.                                                                                                                                                             |
-| ReferencesAndLinks | OPTIONAL              | [array][] of [strings][] | List of references to publication that contain information on the dataset, or links.                                                                                                                                                                  |
-| DatasetDOI         | OPTIONAL              | [string][]               | The Document Object Identifier of the dataset (not the corresponding paper).                                                                                                                                                                          |
+| ReferencesAndLinks | OPTIONAL              | [array][] of [strings][] | List of references to publication that contain information on the dataset, or [URI](../02-common-principles.md#uniform-resource indicator).                                                                                                                                                                  |
+| DatasetDOI         | OPTIONAL              | [string][]               | The Document Object Identifier of the dataset (not the corresponding paper) expressed as a valid [URI](../02-common-principles.md#uniform-resource indicator).                                                                                                                                                                          |
```

Comment:
This could potentially break things for datasets out there that are not using a valid URI for this.

Reply:
agreed, this will most likely break some datasets ... I know for sure that it'll break those that I prepared, see for example https://github.com/bids-standard/bids-examples/blob/8031430b2da07b7f2be58f92b0bd11e2f27a5645/eeg_matchingpennies/dataset_description.json#L15

yet I would be in favor of this breakage for these reasons:

1. it's *very* easy to fix
1. it will lead to a more clear specification (one way to specify the doi, instead of at least 3: `https...`, `doi:...`, or directly `10.17605/OSF.IO/CJ2DR`)
1. it will allow us to better validate this field using the json schemas ... validate a URI formatting instead of simply a string type

Drawbacks:

- Some datasets will remain broken, because their owners won't update them
- We might annoy some users